packages/server-bridge/src/bridge.js

```diff
@@ -63,6 +65,9 @@ function startSession(log, source, socket) {
       log.info(`${session.logPrefix}: ended session`);
       delete sessions[socketId];
     }],
+    setSessionHeaders(headers) {
+      this.headers = assign({}, headers, { bridges: [source] });
```

Comment:
Nice!
If I understand correctly, these headers will be assigned to each request headers, right?

Reply:
And as it is only supports a single bridge, maybe it's better to call it `bridge` and to not support arrays

Reply:
yes & ok